libtextsecure/sendmessage.js

```diff
@@ -663,10 +663,13 @@ MessageSender.prototype = {
     if (!primaryDeviceKey) {
       return Promise.resolve();
     }
-
+    // Extract required contacts information out of conversations
+    const sessionContacts = conversations.filter(
+      c => c.isPrivate() && !c.isSecondaryDevice() && c.isFriend()
+    );
```

Comment:
do we need an `if (sessionContacts.length === 0)` check here?

Reply:
Added
```js
    if (sessionContacts.length === 0) {
      return Promise.resolve();
    }
```